### PR TITLE
[skip-ci] Add tutorial dates if missing, change macro_image to js style

### PR DESCRIPTION
**Pull request by aprozo**
# This Pull request:
Tutorials doxygen documentation:
Add tutorial dates if missing, change macro_image to js style if any drawing methods present (Draw, DrawClone,DrawCopy)

### CODE REVIEW

File: tutorials/gl/glbox.C [hunk 1]
```diff
@@ -2,10 +2,12 @@
 /// \ingroup tutorial_gl
 /// Display a 3D histogram using GL (box option).
 ///
-/// \macro_image(nobatch)
```
Comment (mdessole) [inline tutorials/gl/glbox.C:5]: @couet, could you please confirm that this change is good?

Comment (couet) [reply tutorials/gl/glbox.C:5]: no it is not, \author must be the last directive of the tutorial's headers.

Comment (couet) [reply tutorials/gl/glbox.C:5]: see the last line of this page: https://root.cern/for_developers/docs/guidelines/#tutorials-documentation

**Comment by linev** [comment]
I would split PR on several commit - grouping them by changes in some sub-directory.

And I am not sure that all these tutorials would be possible to run as is.
Sometime one need special data files or one need to have result of some other tutorials.

And do we really want to have all tutorials drawn with JSROOT?

**Comment by linev** [comment]
My doubt is that doxygen generation will never be completed after this PR merged as is.

**Comment by aprozo** [comment]
> I would split PR on several commit - grouping them by changes in some sub-directory.
> 
> And I am not sure that all these tutorials would be possible to run as is. Sometime one need special data files or one need to have result of some other tutorials.
> 
> And do we really want to have all tutorials drawn with JSROOT?

I'll divide it into subdirectories

**Pull request closed without merging**